src/base/static/components/app.tsx

```diff
@@ -272,7 +359,6 @@ class App extends Component {
       }),
     );
 
-    await Promise.all(allPlacePagePromises);
```

Comment:
@goldpbear looks like we were fetching all of places twice... so I removed this.

Type checking helped figure this out :)